src/app/options/options.component.spec.ts

```diff
@@ -93,18 +154,112 @@ describe('OptionsComponent', () => {
     expect((await options[0].getText())).toEqual("test1");
   });
 
+  it('should display mat-errors if input form control is blank', async () => {
+    await createState();
+    const inputError = fixture.debugElement.query(By.css('.input-error'));
+    const outputError = fixture.debugElement.query(By.css('.output-error'));
+    expect(component.inputControl.invalid).toEqual(true);
```

Comment:
These two expects on the component are redundant and can be deleted. Should suffice that inputError and outputError are truthy.